### PR TITLE
Stubs for PyAudio added

### PR DESCRIPTION
**Pull request by jolaf**

### CODE REVIEW

File: stubs/pyaudio/pyaudio.pyi [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+paInputUnderflow: Final[int] = ...
+paInputOverflow: Final[int] = ...
+paOutputUnderflow: Final[int] = ...
+paOutputOverflow: Final[int] = ...
+paPrimingOutput: Final[int] = ...
+
+paMacCoreStreamInfo: PaMacCoreStreamInfo
+
+# Auxiliary types
+ChannelMap = Sequence[int]
```
Comment (JelleZijlstra) [inline stubs/pyaudio/pyaudio.pyi:70]: Prefix these with _ since they don't exist at runtime

File: stubs/pyaudio/METADATA.toml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+version = "0.1"
```
Comment (JelleZijlstra) [inline stubs/pyaudio/METADATA.toml:1]: This means the lowest version that is supported, is that correct? 0.1 is from 2006 (https://pypi.org/project/PyAudio/#history)

**Review by JelleZijlstra — CHANGES_REQUESTED** [review]
Thanks! Two small comments

File: stubs/pyaudio/METADATA.toml (hunk 2 above)
Comment (jolaf) [reply stubs/pyaudio/METADATA.toml]: Changed to the recent version number.

File: stubs/pyaudio/pyaudio.pyi (hunk 1 above)
Comment (jolaf) [reply stubs/pyaudio/pyaudio.pyi]: Done.

**Pull request merged**